docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Knative application under the strict authorization policy"
```

Comment:
It's not clear to me from this title
1. What "strict authorization policy" is and why someone would need it for a Knative app?
2. Whether this section is conceptual information or contains steps to complete a task?

I'd try to make this clearer. Maybe something like:
"Enabling requests to Knative services when additional authorization policies are enabled"

Reply:
I'm also wondering whether this belongs in a more Istio specific place, @mattmoor wdyt?

Reply:
> 1. What "strict authorization policy" is and why someone would need it for a Knative app?

"strict authorization policy" is an Istio's feature. It can control access to allow/deny, which Knative does not have. So someone would like to use it.

> Whether this section is conceptual information or contains steps to complete a task?

It is a "contains steps to complete a task"?

> I'm also wondering whether this belongs in a more Istio specific place

Yes, we definitely should belong in an Istio specific place. But I think I should open a new request to create the new hierarchy of the doc like:
https://knative.dev/docs/serving/istio/some-new-feature (istio)
https://knative.dev/docs/serving/kourier/some-new-feature (kourier)